src/Cake.Prca.Tests/FakeCodeAnalysisProvider.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Cake.Prca.Tests
+{
+    using System.Collections.Generic;
+    using Core.Diagnostics;
+    using Prca.Issues;
+
+    public class FakeCodeAnalysisProvider : CodeAnalysisProvider
+    {
+        private readonly List<ICodeAnalysisIssue> issues = new List<ICodeAnalysisIssue>();
+
+        public FakeCodeAnalysisProvider(ICakeLog log) 
+            : base(log)
+        {
+        }
+
+        public FakeCodeAnalysisProvider(ICakeLog log, IEnumerable<ICodeAnalysisIssue> issues)
+            : base (log)
```

Comment:
No currently analyzers are only in the main project. Introducing StyleCop and other analyzers in the test project would also require a separate ruleset since we would like to enforce only a small subset of the rules for test assemblies.